src/main/java/com/kobylynskyi/graphql/codegen/model/definitions/ExtendedEnumTypeDefinition.java

```diff
@@ -0,0 +1,23 @@
+package com.kobylynskyi.graphql.codegen.model.definitions;
+
+import graphql.language.EnumTypeDefinition;
+import graphql.language.EnumTypeExtensionDefinition;
+import graphql.language.EnumValueDefinition;
+
+import java.util.List;
+
+public class ExtendedEnumTypeDefinition extends ExtendedDefinition<EnumTypeDefinition, EnumTypeExtensionDefinition> {
+
+    /**
+     * Get enum value definitions from the definition and its extensions
+     *
+     * @return list of all enum value definitions
+     */
+    public List<EnumValueDefinition> getValueDefinitions() {
+        List<EnumValueDefinition> definitions = definition.getEnumValueDefinitions();
+        extensions.stream()
+                .map(EnumTypeExtensionDefinition::getEnumValueDefinitions)
+                .forEach(definitions::addAll);
```

Comment:
I'm not sure how safe this is. Is `definition.getEnumValueDefinitions()` creating a defensive copy of the list internally?
To be on the safe side, I'd rather create a list on the spot add stuff from the base definition and extensions.

Reply:
Yes, they are creating a defense copy.